### ISSUE TITLE
Some mss and msParts have multiple classification codes

### ISSUE THREAD

**Issue by wlpotter** [srophe/britishLibrary-data#1104]
Each single-part ms or the msParts of multi-part mss should have one Wright classification code.

Examples include https://bl.vuexistapps.us/manuscript/75.

Search for the following to find cases of multiple: 

`name="dcterms:type"` in Xpath: `//listRelation/relation[position() > 1]`. Found 24 instances; most likely just encoding errors?

**Comment by wlpotter** [comment]
Cf. #120.

It's possible that in transition from the profileDesc to the relations in tei:head that some errors were introduced?

**Comment by wlpotter** [comment]
Might be relevant to schema changes if we decide to limit the number of relations here

**Comment by wlpotter** [comment]
A few notes on this one to discuss:

- Some came from multi-layered parts (e.g., 106.xml, part 1 (107) has a wright entry that we subdivided into sub-parts) that ended up duplicating classifications. Removed extra but kept the bible-ot for 107 and its sub-parts
    - changed the 1501 subpart to be unclassified since it's a flyleaf we identified
- 143.xml part 4 has two classifications: theo-collected based on location in catalogue and bible-nt that seems to have been added by someone b/c there are NT texts. I think we remove and only have where the entry is located. Connection to NT would come based on connection to the work record, which has an NT genre. But this will be a later functionality
